bodhi/server/services/updates.py

```diff
@@ -92,6 +94,12 @@ def get_update(request):
                  error_handler=bodhi.server.services.errors.html_handler)
 def get_update_for_editing(request):
     """Return a single update from an id, title, or alias for the edit form"""
+    proxy_request = bodhi.server.security.ProtectedRequest(request)
+    validate_acls(proxy_request)
+    # If validate_acls produced 0 errors, then we can edit this update.
+    can_edit = len(proxy_request.errors) == 0
+    if not can_edit:
+        raise HTTPForbidden()
```

Comment:
I think we could probably accomplish this by using the Pyramid ACL system. There is some code in the definition of ```update_edit``` near the top of this file that I would think should do this, but I suppose something is wrong there:

```
 53 update_edit = Service(
 54     name='update_edit', path='/updates/{id}/edit', validators=(validate_update_id,),
 55     description='Update submission service', acl=bodhi.server.security.packagers_allowed_acl,
 56     cors_origins=bodhi.server.security.cors_origins_rw)
```

I'd expect that ```acl=``` line to do the trick, but I'm not sure why it's not working. I think we should focus on fixing it that way so we're doing it "the Pyramid way".